examples/grpc-client-mutual-ssl/grpc_client_mutual_ssl.md

```diff
@@ -2,15 +2,33 @@
 
 Ballerina supports mutual SSL, which is a certificate-based authentication
 process in which two parties (the client and server) authenticate each other by
-verifying the digital certificates. It ensures that both parties are assured
-of each other's identity.<br/><br/>
+verifying the digital certificates. It ensures that both parties are assured of each other's identity.
+
 For more information on the underlying module, 
-see the [gRPC module](https://docs.central.ballerina.io/ballerina/grpc/latest/).
+see the [gRPC module](https://lib.ballerina.io/ballerina/grpc/latest).
 
 ::: code grpc_client.proto :::
 
+Create a new Protocol Buffers definition file named `grpc_client.proto` and add the service definition to it.
+Run the command below in the Ballerina tools distribution for stub generation.
+
+Once you run the command, `grpc_client_pb.bal` file is generated inside stubs directory.
+
+For more information on how to use the Ballerina Protocol Buffers tool, see the <a href="https://ballerina.io/learn/by-example/proto-to-ballerina.html">Proto To Ballerina</a> example.
+
 ::: out grpc_client.out :::
 
 ::: code grpc_client_mutual_ssl.bal :::
 
-::: out grpc_client_mutual_ssl.out :::
\ No newline at end of file
+Create a Ballerina package.
```

Comment:
```suggestion
Create a Ballerina package.
```
We can make these steps numbered bullet points. Check and fix all the occurrences.